### PR TITLE
Cleaner Grenade

### PR DESCRIPTION
**Pull request by DenisShvalov**
Added Cleaner Grenade

## About the PR
Added Cleaner Grenades that creates foam of space cleaner of activate
- Added 2 grenades in janitor belt on spawn
- Added 2 grenades in custodial closet
- Added 2 grenades in janitorial supplies crate

## Why / Balance
Big puddles of space lube 💀 

## Media

https://github.com/space-wizards/space-station-14/assets/115770678/99fe98e1-c8fa-49d4-8658-a73519887606

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- add: Added Cleaner Grenades that will help janitors in their work

### CODE REVIEW

**Comment by 0x6273** [comment]
You can already make these with modular grenade chem payload.

**Comment by DenisShvalov** [comment]
> I think it'd be cooler if you crafted them from like soap and cloth or something. Since buying a whole supply crate for 2 grenades is a bit wasteful and the kitchen can make soap, more interaction is always better.

This grenade can by created by chemist with 50 fluoro, 50 water and 30 space cleaner. Craft with soap and cloth is to op in my sight

**Comment by Golinth** [comment]
I support this. As a nukie, or syndie chemist, I tend to always pack a lube foam grenade or two, and this will certainly help prevent how OP that strategy is.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Remove it from the locker as it's not essential to the job.

**Comment by DenisShvalov** [comment]
> Remove it from the locker as it's not essential to the job.

Why not? Janitors can fast clean large puddles with them, so they will use them often.
I decided to add them to lockers, because in ss13 you can find them in jani lockers

**Comment by Catdere** [comment]
> > Remove it from the locker as it's not essential to the job.
> 
> Why not? Janitors can fast clean large puddles with them, so they will use them often. I decided to add them to lockers, because in ss13 you can find them in jani lockers

I agree with this, As having a few cleaner grenades in your locker can help keep the station clean until sci can hit you with a better mod

**Comment by metalgearsloth** [comment]
> > Remove it from the locker as it's not essential to the job.
> 
> Why not? Janitors can fast clean large puddles with them, so they will use them often. I decided to add them to lockers, because in ss13 you can find them in jani lockers

The locker is so someone can take that role if they change their job; the grenade is not essential to the job.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
In case my comment wasn't clear put it on a vendor or equivalent, lockers are supposed to be barebones and not just a mess of sometimes roundstart stuff sometimes not.

File: Resources/Prototypes/Catalog/Fills/Items/belt.yml [hunk 1]
```diff
@@ -63,8 +63,8 @@
     contents:
       - id: Soap #Make a soap group and pick between when i'm not lazy
       - id: SprayBottleSpaceCleaner
-      #- name: GrenadeChem
-      #- name: GrenadeChem
+      - id: CleanerGrenade
+      - id: CleanerGrenade
       - id: FlashlightLantern
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Items/belt.yml:68]: Why isn't this using amount.

**Comment by DenisShvalov** [comment]
Removed from jani locker, added box of them that you can buy in cargo with jani items

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**